nginx-webserver-01/nginx-webserver-01.dockerapp

```diff
@@ -0,0 +1,14 @@
+version: 1.0.0
+name: nginx-webserver-01
+description: Foundries.io nginxwebserver-01 example
+
+version: '1.0'
+
+services:
+  nginx:
+    image: hub.foundries.io/<your factory>/ngins-webserver-01:latest
```

Comment:
Here you got 'your factory' as part of the link, which I believe is expected since you are using this app as an example.